src/types/swiper-events.d.ts

```diff
@@ -223,6 +223,14 @@ export interface SwiperEvents {
    * INTERNAL: Event will fired after setting CSS classes on swiper slide element
    */
   _slideClass?: (swiper: Swiper, el: HTMLElement, classNames: string) => void;
+
+  /**
+   * INTERNAL: Event will fired after setting CSS classes on all swiper slides
+   */
+  _slideClasses?: (
+    swiper: Swiper,
+    slides: { el: HTMLElement; classNames: string; index: number }[],
+  ) => void;
```

Comment:
I use it in angular to rerender class names. So I think it will be better to rerender it once, instead of on every slide.